algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -38,23 +38,21 @@ package com.twitter.algebird
  */
 
 object QTree {
-  def apply[A: Monoid](kv: (Double, A), level: Int = -16): QTree[A] = {
+  def apply[A](kv: (Double, A), level: Int = -16): QTree[A] =
     QTree(math.floor(kv._1 / math.pow(2.0, level)).toLong,
       level,
       1,
       kv._2,
       None,
       None)
-  }
 
-  def apply[A: Monoid](kv: (Long, A)): QTree[A] = {
+  def apply[A](kv: (Long, A)): QTree[A] =
     QTree(kv._1,
       0,
       1,
       kv._2,
       None,
       None)
-  }
 
   /**
    * The common case of wanting a count and sum for the same value
```

Comment:
Yes, though I think `offset` is the wrong name (and that in fact it's totally fine to have that be `apply`)